src/main/323C.c

```diff
@@ -2790,21 +2825,45 @@ s32 func_8002D25C(struct PlayerObj* arg0)
     }
 }
 
-INCLUDE_ASM("asm/us/main/nonmatchings/323C", func_8002D32C);
+s32 func_8002D32C(struct PlayerObj* arg0, s16 arg1, s32 arg2)
+{
+    if (func_8002D41C(arg0, func_8002D724(arg0, D_8013B7F8, arg1), arg2) == 0) {
```

Comment:
This also matches, either way is fine https://decomp.me/scratch/KA5p0

Reply:
With `./format` this becomes:

```c
if (func_8002D41C(arg0, func_8002D724(arg0, D_8013B7F8, arg1), arg2) == 0 && func_8002D41C(arg0, func_8002D724(arg0, D_8013B7F8 - D_8013B7E0, arg1), arg2) == 0 && func_8002D41C(arg0, func_8002D724(arg0, D_8013B7F8 + D_8013B7E0 - 1, arg1), arg2) == 0) {
```

Reply:
Ok, either adjusting the .clang-format or keeping the current way is fine with me

Reply:
I couldn't figure out how to fix clang-format without setting `ColumnLimit`, going to leave it as-is for now.